mapie/calibration.py

```diff
@@ -0,0 +1,502 @@
+from typing import Optional, Union, cast, Tuple, Dict
+
+import warnings
```

Comment:
warnings should be imported with typing as they are both python packages

Reply:
done. (isort does it automatically! :))